### DISCUSSION TITLE
请问怎么让检测结果的阈值设置为0.5（默认0.50:0.95）？

### DISCUSSION THREAD

**Discussion by lyk595** [open-mmlab/mmdetection#9043] [Q&A] [answered]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 💬 Describe the reimplementation questions

请问怎么让recall的检测结果的阈值设置为0.5（默认0.50:0.95）？precision默认是既有0.5，也有0.5:0.95，要怎么让recall像precision这样？

### Environment

请问怎么让recall的检测结果的阈值设置为0.5（默认0.50:0.95）？precision默认是既有0.5，也有0.5:0.95，要怎么让recall像precision这样？

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by taofuyu** [comment, marked as answer]
evaluation = dict(interval=1, metric='bbox', classwise=True, metric_items=['mAP_50', 'AR@100'], iou_thrs=[0.5])
